automated-comments/ruby/two-fer/string_building.md

```diff
@@ -1 +1,9 @@
-In Ruby, it is unusual to use string addition. People tend to use `sprintf`, `format`, `%%`, or string interpolation. So instead of `"One for " + %<name_variable>s + ", one for me."`, consider `"One for #{%<name_variable>s}, one for me."`.
+String concatenation is perfectly valid within Ruby, but a more common way to compose a string is with [string interpolation](http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html). For example:
+
+```ruby
```

Comment:
Isn't giving the solution going against the suggested automated-mentoring guidelines: https://github.com/exercism/automated-mentoring-support/pull/61/files#diff-c3fc7e01c1c858a88579c56982ac5ba5R60?